openfl/interface/aggregator.py

```diff
@@ -90,6 +91,21 @@ def start_(plan, authorized_cols, task_group):
             parsed_plan.config["assigner"]["settings"] = {}
         parsed_plan.config["assigner"]["settings"]["selected_task_group"] = task_group
         logger.info(f"Setting aggregator to assign: {task_group} task_group")
+    
+    # Check if remote attestation is enabled in the plan configuration
+    if parsed_plan.config["network"]["settings"].get("enable_remote_attestation", False):
+        # Fetch remote attestation environment variables
+        env_vars = attestation_utils.fetch_attestation_env_vars()
+        if env_vars is not None:
+            attestation_mr = attestation_utils.AttestationManager("aggregator", env_vars["ATTESTATION_REPORT_PATH"], env_vars["ITA_API_KEY"], env_vars["AVS_URL"])
+            # Generate and store the attestation report
+            attestaion_report = attestation_mr.get_attestation_report()
```

Comment:
The `attestation_report` object would only be used in OpenFL-Security, so here I suggest simply doing `attestation_mr.get_attestation_report()`.

Reply:
done